src/user-research/taking-notes-and-recording-research-sessions.md

```diff
@@ -21,116 +17,107 @@ Here we provide additional, specific guidance for researchers at dxw.
 
-   Keep only anonymised notes, quotes and images in tools like Miro and Trello.
-4. **Have a fall back**
+We make sure the files
+[have good names](/user-research/taking-care-of-research-data/#identifying-research-data-clearly),
+so we can find them and delete them when we need to.
 
-   The one certainty in user research is that things go wrong. So make sure you
-   have other options to capture what you need.
+We keep only anonymised notes, quotes and images in tools like Miro and Trello.
 
-   For example:
+### Do not use our personal devices
 
-   * The participant may decide they don’t want to be recorded in a particular
-     way, so be prepared to switch from video to an audio-only recording, or to
-     just take notes
-   * Your preferred device, app or service may not be working properly, so have
-     an alternative like a paper notebook
-   * Your note taker, assistant or observer may not be able to join the session,
-     so be prepared to take notes and recordings yourself
+We always use a
+[dxw provided device]](#using-a-dxw-provided-handheld-device),
+rather than our personal device, when taking notes and photos, or making recordings.
 
-   Note that if the participant refuses all recording, including note taking,
-   you may want to abandon the session.
+Using our own devices in research means that personal data about research participants can be mixed into our personal data stores, like cloud photo libraries. And also runs the risk of participants, client staff and colleagues seeing our personal messages and images.
 
-## Using your dxw provided laptop
+### Have a fall back
 
-Whenever possible, use your dxw provided laptop for taking notes and making
-recordings. Our laptops have encrypted drives and we follow good security
-policies.
+The one certainty in user research is that things go wrong. So we make sure we have other options to capture what we need.
 
-With your laptop you can:
+For example:
 
-* capture notes to a local file or a Google Sheet
-* record audio using a microphone and the Voice Memos or Quicktime apps
-* record video using an external camera and Quicktime
-* record your laptop screen using Quicktime or Screenshot
+* The participant may decide they don’t want to be recorded in a particular way, so we are prepared to switch from video to an audio-only recording, or to just take notes
+* Our preferred device, app or service may not be working properly, so we have an alternative like a paper notebook
+* Our note taker, assistant or observer may not be able to join the session, so we are prepared to take notes and recordings ourselves
 
-If research participants might use your laptop or see your screen during a
-research session, it’s a good idea to create a separate ‘research’ user account
-on your laptop with things like notifications and browser autofill turned off.
+If our participant refuses all recording, including note taking, we accept that we might need to abandon the session.
```

Comment:
Joh, this feels one sided to one scenario, are we then advised to not go ahead with a research session when there is not recording consent of any type?
I would say no, that we should go ahead with a session if we are experienced enough take quick notes and reflective notes after the session. For instance, at the Department for Education I had about 4 participants that refused recording of any type, there were not takers but no recording. I still went ahead with the session and it was useful for the research project.

Reply:
My intention was to give researchers permission to abandon a session, rather than to say they had to.
In the past I've seen researchers feel like that had to complete a session no matter what happened.

Would this be clearer:
If our participant refuses all recording, including note taking, we can continue with a session. But accept that we might need to adapt or abandon the session.

Reply:
If your intention is to give user researchers permission to abandon the session then just say so.
I think that trying to say something without saying it, is not helpful.
Here it is compounded because there are to messages: they one you are trying to say but not saying and the fact that some user researchers level of experience will mean it is best for them to run or not run the session.

This section is about taking notes and recording research sessions. Do you think that there is another section in the playbook where we can say or we are already saying explicitly that user researchers can opt to end the session for whatever reason (distress, competency when video/audio recording is not consented, other...)?

It would be clearer to say then: If our participant refuses all recording, including note takers, we might choose to continue with the session or cancel it.

Note how I have referred explicitly to including 'note takers' and not taking notes, because, are we introducing a research step in which we ask participants if we as user researchers can take notes APART from having note takers?

Reply:
I have meetings now. I'll think about those comments, and respond tomorrow.

Reply:
Following our discussion, I've changed this to:

We can decide to cancel a session if essential note taking, recording or other data collection is not possible. Whether because of technical problems, or because the participant does not consent.